src/View/InertiaJsonView.php

```diff
@@ -47,7 +47,12 @@ public function render(?string $view = null, $layout = null): string
      */
     private function getCurrentUri(): string
     {
-        return Router::url($this->getRequest()->getRequestTarget(), true);
+        $url = Router::url($this->getRequest()->getRequestTarget(), true);
+        if ($this->getRequest()->scheme() === 'https') {
+            $url = str_replace('http','https', $url);
```

Comment:
it could make urls containing more than 1 http to fail, would it be better to place the scheme directly at the beginning and remove it from the requestTarget. There are also other ways to replace the scheme, check bootstrap.php file

Reply:
solved! problem was resolved configuring App.fullBaseUrl